components/tree/private/branch.jsx

```diff
@@ -138,6 +138,11 @@ const handleKeyDownLeft = (event, props) => {
 		);
 		const index = nodes.indexOf(props.parent);
 		if (index !== -1) {
+			props.onExpandClick(event, {
```

Comment:
Hitting left on a branch that's already collapsed should collapse the parent branch and focus the parent node, like how it works for collapsing an item.